src/app/newSettings/Sections/Account/Plans/components/PlanCard.tsx

```diff
@@ -32,6 +32,24 @@ const PlanCard = ({
 }: PlanCardProps) => {
   const userText = isBusiness ? '/' + t('preferences.account.plans.user') : '';
 
+  const planTypeTextPath = (() => {
+    if (capacity === '1TB') {
+      if (!isBusiness) {
+        return 'essentialFeatures';
+      } else {
+        return 'standardFeatures';
+      }
+    } else if (capacity === '2TB') {
+      return 'proFeatures';
```

Comment:
missing the translation in translations files for `proFeatures` in individual plans